api/src/services/script-runner.service.ts

```diff
@@ -63,6 +63,11 @@ export class ScriptRunnerService {
    *
    * @param amiChartImportDTO this is a string in a very specific format like:
    * percentOfAmiValue_1 householdSize_1_income_value householdSize_2_income_value \n percentOfAmiValue_2 householdSize_1_income_value householdSize_2_income_value
+   *
+   * Copying and pasting from google sheets will not match the format above. You will need to perform the following:
+   * 1) Find and delete all instances of "%"
+   * 2) Using the Regex option in the Find and Replace tool, replace /\t with " " and /\n with "\\n"
+   * See "How to format AMI data for script runner import" in Notion for a more detailed example
```

Comment:
I'm wondering if we should be referencing notion in the code base. Especially since at some point we will no longer own the doorway code base